### DISCUSSION TITLE
’RADIOMETER GAS’ command

### DISCUSSION THREAD

**Discussion by d-pit** [firemodels/fds#13769] [Q&A] [answered]
Hello community,

I need your advice on using the ’RADIOMETER GAS’ command. I tried to start a simulation with this command in FDS-6.9.1-0-g889da6a-release version, but it does not work, and this error occurred: ERROR(101): Problem with DEVC number 124

I wrote the code as follows: 
&DEVC ID='RADG1-100', QUANTITY='RADIOMETER GAS’, PROP_ID = 'R', XYZ=0.22,0.7,0.5, ORIENTATION=0.0,1.0,0.0/
where PROP ID is:
&PROP ID = 'R',
	GAUGE_EMISSIVITY = 0.95,
	GAUGE_TEMPERATURE = 20.0/

What did I do wrong? If I change the quantity to, for example, 'GAUGE HEAT FLUX GAS', the code works, and the simulation starts. I also tried reinstalling FDS, but it did not help.

Thank you for your advice.

**Comment by drjfloyd** [comment]
Please attach a simple version of your input file that demonstrates the problem.

**Comment by mcgratta** [comment, marked as answer]
My guess is that you probably used a text editor that added a non-ASCII font or you cut and pasted from Word or something like that. Can you spot the problem?

![image](https://github.com/user-attachments/assets/09a426f8-38aa-4f5e-9a1c-c7a681a08b13)

**Reply by d-pit** [reply]
Omg, yes, that was the problem. Sorry for this stupid question, which was my mistake. I use Notepad for editing codes. I copied the command with the last apostrophe directly from the user guide (to not write it wrong :D). I've checked it, and when I make a copy from the user guide, it makes this:
![wrong](https://github.com/user-attachments/assets/dfe886b7-0995-4ed8-9aac-f69e7c6b5e8e)
instead of this:
![correct](https://github.com/user-attachments/assets/eaad80fb-7ca2-4f67-9376-ee524de952d0)

Thank you for solving that.

_Edit: I use Adobe Acrobat Standard to open the PDFs._

**Reply by mcgratta** [reply]
In our manuals, which are typeset using LaTeX, we use a so-called "verbatim" style for demonstrating input file formats. I'm not sure why the pdf distiller changes the single quote to an end-quote. Something to ponder.

I use the old vi editor, which is less likely to introduce non-ASCII characters.

**Reply by mcgratta** [reply]
As always, we are not the first to discover [this problem](https://www.ctan.org/tex-archive/macros/latex/contrib/upquote).

**Reply by drjfloyd** [reply]
Found this on texstackexchange:

\usepackage{upquote}

{\ct \textquotesingle RADIOMETER \textquotesingle}

![image](https://github.com/user-attachments/assets/3bd3bd23-6369-4b78-9b94-f6d85bb5eee3)

**Reply by mcgratta** [reply]
This works, but it would be tedious to make all these substitutions. I'm trying to redefine `\ct` but each attempt has negative side effects.

**Reply by drjfloyd** [reply]
Unfortunately trying to do set of global search and replace (make all single quotes \textquotesingle and then search/replace to recover possessive nouns and double end quotes) doesn't work since the lstlisting blocks don't process the \textquotesingle and just print it as is.  There is something like 5500 single qouotes so be a couple hours to do a global search and replance checking one by one.

**Reply by mcgratta** [reply]
I've been sucked in. I think I have a solution using `\lstinline`, but there are still things to fix.
